pkg/clients/iam/iam_client.go

```diff
@@ -110,12 +106,12 @@ func (client *IAMClient) AttachS3WildcardInlinePolicy(ctx context.Context, userN
 	}
 
 	_, err := client.IAMService.PutUserPolicy(ctx, input)
-	if err != nil {
-		return fmt.Errorf("failed to attach inline policy to IAM user %s: %w", userName, err)
-	}
+	// if err != nil {
```

Comment:
Can you remove the comment completely ?

Reply:
Oh yes, I will do that, its a left over.